pkg/controller/unifiedpushserver/unifiedpushserver_controller.go

```diff
@@ -52,7 +53,17 @@ func Add(mgr manager.Manager) error {
 
 // newReconciler returns a new reconcile.Reconciler
 func newReconciler(mgr manager.Manager) reconcile.Reconciler {
-	return &ReconcileUnifiedPushServer{client: mgr.GetClient(), config: mgr.GetConfig(), scheme: mgr.GetScheme()}
+	clientset, err := kubernetes.NewForConfig(mgr.GetConfig())
+	if err != nil {
+		log.Error(err, "Failed to get clientset")
+		os.Exit(1)
```

Comment:
I don't know what to do here for this error case. Is it okay to completely bail like this?

This shouldn't ever be hit in any case, and if it is, I don't know how to continue, so exiting seems fair to me.